dev_guide/deployments.adoc

```diff
@@ -131,12 +152,12 @@ shows deployments in the *Browse* tab.
 To cancel a running or stuck deployment:
 
 ----
-$ oc deploy <deployment_config> --cancel
+$ oc deploy --cancel dc/<name>
 ----
 
 WARNING: The cancellation is a best-effort operation, and may take some time to
-complete. It's possible the deployment will partially or totally complete
-before the cancellation is effective.
+complete. It's possible the replication controller will partially or totally complete
+being deployed before the cancellation is effective.
```

Comment:
then I don't understand what is the sentence above describing... if cancelled in a middle of deployment, it will rollback but the above says something like "if cancelled in a middle of deployment, you might end up with half-scaled replication controller"